lib/Cake/Test/Case/Network/Email/CakeEmailTest.php

```diff
@@ -472,6 +468,33 @@ public function testFormatAddress() {
 	}
 
 /**
+ * Test that addresses are quoted correctly when they contain unicode and
+ * commas
+ *
+ * @return void
+ */
+	public function testFormatAddressEncodeAndEscape()
+	{
```

Comment:
@markstory it's CakePHP 2.x, not 3.x :trollface:  refs build error https://travis-ci.org/cakephp/cakephp/jobs/242248221#L246